cdhweb/people/urls.py

```diff
@@ -16,5 +16,4 @@
     url(r'^faculty/$', RedirectView.as_view(url='/people/affiliates/', permanent=True)),
     # redirect from /people/postdocs -> /people/staff
     url(r'^postdocs/$', RedirectView.as_view(url='/people/staff/', permanent=True)),
-    url(r'^(?P<slug>[\w-]+)/$', views.ProfileDetailView.as_view(), name='profile'),
```

Comment:
Removed from url config because now handled by wagtail page routing?

Do we need to do anything about that? (Anything to be aware of or note as we move forward with the wagtail exodus?)

Reply:
correct. eventually i think the only thing in these urlconfs will be sitemaps, using wagtail's contrib.sitemap module. the redirects we can convert into ones stored in the database, so that they can be changed/managed in the admin.